### PR TITLE
Add playerflags, dnsbl, and ips for non-admins in status

### PR DESCRIPTION
**Pull request by 12pm**

### CODE REVIEW

File: src/engine/server/server.cpp [hunk 1]
```diff
@@ -407,6 +408,15 @@ void CServer::SetClientScore(int ClientID, int Score)
 	m_aClients[ClientID].m_Score = Score;
 }
 
+void CServer::SetClientFlags(int ClientID, int Flags)
+{
+	if(ClientID < 0 || ClientID >= MAX_CLIENTS || m_aClients[ClientID].m_State < CClient::STATE_READY)
+		return;
+
+	if(Flags > m_aClients[ClientID].m_Flags)
```
Comment (def-) [inline src/engine/server/server.cpp]: why the if here?

**Comment by def-** [comment]
Rest looks ok.

File: src/engine/server/server.cpp (hunk 1 above)
Comment (12pm) [reply src/engine/server/server.cpp]: To track the highest rather than the current one, more useful I think

**Comment by def-** [comment]
LGTM, I'll wait for some other admin to decide if we want this.

**Comment by fokkonaut** [comment]
IPs for non admin in status? Thats dumb, that got removed because it got abused on public test servers

**Comment by 12pm** [comment]
No? Normal players don’t even have(need) access to status on test server..

**Comment by fokkonaut** [comment]
still, why does a helper need ips?

**Comment by def-** [comment]
So that a helper can report to us on Discord that some guy keeps joining with IP X.X.X.X with different names blocking people constantly. What's wrong about knowing IPs?

**Comment by fokkonaut** [comment]
on test servers people used status to get ips to ddos.

**Comment by def-** [comment]
So we're not supposed to trust our own staff? Then maybe get rid of mods/helpers entirely.

**Comment by heinrich5991** [comment]
The problem was that it was accidentally published via a stream.

IP addresses should not be visible by default, IMO. E.g. Discourse always hides them behind another separate click.

**Comment by def-** [comment]
So how about a command in f2 to enable IP display for your own connection only and making all commands check that setting being on and then displaying the actual IP or an empty placeholder?

**Comment by heinrich5991** [comment]
Sounds good.

**Comment by def-** [comment]
@12pm Do you want to add that or should I?

**Comment by 12pm** [comment]
@def- you can do it if you want

**Comment by def-** [comment]
Hm, actually that's not so easy. Right now the same console text is sent to everyone logged into rcon.

**Comment by def-** [comment]
https://github.com/ddnet/ddnet/pull/1571

**Pull request merged**